### DISCUSSION TITLE
Links in wiring diagram PDF are not working

### DISCUSSION THREAD

**Discussion by ph0enix4** [sumatrapdfreader/sumatrapdf#3227] [Q&A]
Hello,

I'm using the last version of Sumatra 3.4.6 (64 bits) on Windows 10 64 bits.
I see that links are not working in this PDF.
It works with Adobe reader and also with Chorme PDF viewer.

For example go on page 18/53 and try to click on "15.2E" or "/50.3B". It will work with Adobe or Chrome but not with Sumatra.
Hope you can solve it. :)

Link for the PDF https://www.swisstransfer.com/d/fe5efcb3-a7ad-47d9-9019-29f17997ed0a 

Best regards.

**Comment by GitHubRulesOK** [comment]
Hmm cant see any links visually perhaps they are javascript (which SumatraPDF does not support) Chromium is able to run JS without asking
![image](https://user-images.githubusercontent.com/12754913/212780767-ed3b887e-6372-45b0-8cdf-5af772f631b7.png)
![image](https://user-images.githubusercontent.com/12754913/212781857-1f6ea2b5-d4b3-448b-8a6f-6fc40ada89f1.png)

**Comment by ph0enix4** [comment]
Thanks for your quick reply.

I tried with Foxit and it works well.
So, you mean Adobe reader and Foxit Reader can use javascript ? Any chance to see javascript on Sumatra ? 

PS: please delete the file, don't upload it to github !

**Reply by GitHubRulesOK** [reply]
link removed but the only reason I devote so much time to SumutraPDF is it does NOT use JavaScript otherwise I may as well use fast lesser secured Edge as native to Windows.
